Assets/MRTK/SDK/Experimental/Features/UX/BoundsControl/Visuals/PerAxisHandles.cs

```diff
@@ -0,0 +1,309 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using Microsoft.MixedReality.Toolkit.Experimental.UI.BoundsControlTypes;
+using UnityEngine;
+using System.Collections.Generic;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.UI.BoundsControl
+{
+    /// <summary>
+    /// Rotation handles for <see cref="BoundsControl"/> that are used for rotating the
+    /// Gameobject BoundsControl is attached to with near or far interaction
+    /// </summary>
+    public abstract class PerAxisHandles : HandlesBase
+    {
+        protected override HandlesBaseConfiguration BaseConfig => config;
```

Comment:
For any of these that are abstract/not implemented here, it would be good to have doccomments on them.

i.e. in general anytime you have something that needs to be implemented by a subclass, having the intention of the function and expectations documented are key for making sure that implementors know to do the right thing.